src/components/app/components/App.js

```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Layout } from 'antd';
 
+import home from '../../home';
 import Footer from './Footer';
 import Navbar from './Navbar';
 
+const { ContactUs } = home.components;
```

Comment:
is it possible for you to pull the `ContactUs subComponent out into its own component. I think it warrants it given that it has its own redux logic?